3-enrich/scala-common-enrich/build.sbt

```diff
@@ -25,6 +25,7 @@ lazy val root = project
   .settings(BuildSettings.formatting)
   .settings(BuildSettings.buildSettings)
   .settings(BuildSettings.publishSettings)
+  .settings(parallelExecution in Test := false)
```

Comment:
Tests running locally in SBT hung in my computer. The only thing that make them pass is to run them sequentially. Maybe you don't see it because Travis is too slow and the race condition doesn't manifest. I will remove it from the PR anyway.